src/demo/components/ImageSlider.css

```diff
@@ -138,3 +138,88 @@ line-height: 35px !important;
   letter-spacing: 0.5px; 
   text-decoration: none;
 }
+.ellipse-container {
+  display: flex; 
+  justify-content: center; 
+  align-items: center; 
+  margin-top: 20px; 
+}
+
+.ellipse{
+  width: 18px; 
+  height: 18px; 
+  border: 2px solid var(--white) ;
+  border-radius: 50%;
+  margin: 0 10px;
+  cursor: pointer;
+}
+.active-ellipse {
+  background-color: var(--white);
+
+}
+@media only screen and (min-width: 375px) and (max-width: 767px) {
+  .slider-container {
+    flex-direction: column;
+    overflow-x: hidden;
+    overflow-y: hidden;
+    margin-top: 2rem;
+
+  }
+
+  .center-image {
+    padding: 0;
+    max-width: 100%;
+    text-align: center;
+    min-width: 90%;
+    margin-left: -2.5rem;
+  }
+  .left-side-image,
+  .right-side-image {
+    text-align: center;
+    max-width: 100%;
+  }
+
+  .left-side-image img,
```

Comment:
we should use hyphon-separated-text or camelCase for classes, but not both. Please choose one and stick to it.